native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -572,7 +567,7 @@ JNIEXPORT void JNICALL Java_sage_Sage_addTaskbarIcon0(JNIEnv *env, jclass jc, jl
 	iconData.hIcon = LoadIcon(GetModuleHandle(NULL), MAKEINTRESOURCE(iconID));
 	static jclass sagetvClass = (jclass) env->NewGlobalRef(env->FindClass("sage/SageTV"));
 	env->ExceptionClear();
-	strcpy(iconData.szTip, sagetvClass ? "SageTV 2" : "SageRecorder");
+	strcpy_s(iconData.szTip, sizeof(iconData.szTip), sagetvClass ? "SageTV 2" : "SageRecorder");

```

Comment:
Just make the 'SageTV 2' be 'SageTV'...that clearly hasn't been updated since 2004. :)

Reply:
Done